spec/notifications_spec_helpers.rb

```diff
@@ -0,0 +1,16 @@
+module NotificationsSpecHelpers
+  def create_consent_change(new_consent, recipient, program, datetime)
```

Comment:
Was there a reason this could not be done with `FactoryBot`?

Reply:
Is there a way to create the dependency (SmsMessage) directly from the ConsentChange factory bot constructor? I only broke this out because I didn't know how to do that.

Reply:
https://github.com/codeforamerica/notifications/blob/main/spec/factories/recipient.rb#L3